From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 26 Apr 2016 23:12:20 +0200
Subject: [PATCH] f_repobuild/Makefile: Use python-debian from pip.

Currently, Ubuntu Trusty provides a broken python-debian package,
that fails to parse foreign architecture (package:any) relationships.

Purge debian package and let pip resolve this dependency, using a
newer version from pypi.

[1] http://lists.alioth.debian.org/pipermail/
    pkg-python-debian-maint/2014-July/001795.html
---
 build/f_isoroot/f_repobuild/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build/f_isoroot/f_repobuild/Makefile b/build/f_isoroot/f_repobuild/Makefile
index 3d1b566..36d6c8e 100644
--- a/build/f_isoroot/f_repobuild/Makefile
+++ b/build/f_isoroot/f_repobuild/Makefile
@@ -25,6 +25,8 @@ all: nailgun
 
 nailgun:
 	sudo apt-get install -y git libxml2-dev libxslt-dev python-dev  python-pip libz-dev libyaml-dev createrepo python-yaml
+	# python-debian from Ubuntu can't parse foreign arch relationships, use pip
+	sudo apt-get purge -y --force-yes python-debian
 	rm -Rf nailgun
 	sudo mkdir -p /var/www/nailgun
 	git clone $(FUEL_MIRROR_REPO)
